.github/workflows/publish_on_dockerhub.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    tags:
+      - 'release/*'
+
+name: Release
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  create_release:
```

Comment:
As far as I can see this doesn't create a github release? Why is this included? 🙂

Reply:
btw, I think having github releases as well is a really good idea :)

Reply:
this was just for version parsing, isn't it necessary?